exercises/arrays/lodashCompact/index.ts

```diff
@@ -0,0 +1,6 @@
+const arrTask2: any[] = [0, 1, false, 2, "", 3];
+
+const compact = (arrTask2: any[]): any => arrTask2.filter(Boolean);
```

Comment:
Try to improve typing here. 
https://www.typescriptlang.org/docs/handbook/2/generics.html
https://www.typescriptlang.org/docs/handbook/utility-types.html#omitthisparametertype

Reply:
I didn't completely understand what to do with `OmitThisParameter<Type>`, so I suggest adding here a new data type `MyArrayType` that will include types allowed for this array.